### DISCUSSION TITLE
为闭源插件&主题提供集中反馈issue的仓库

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [halo-dev/halo#6969] [Ideas]
期望为应用市场一些闭源付费插件&主题提供一个反馈issue的仓库。
虽说现有的反馈位置是应用市场的评论区,但是毕竟那个地方基本没人看,实用性也实在不如GitHub。
故提出此想法。

**Comment by QingFeng-awa** [comment]
目前大部分用户是在halo-dev的讨论区发布一些功能请求,BUG反馈,有些甚至直接在halo仓库发issue。
我认为这样管理不够高效,也会增加组织管理者的工作负担(因为看样子在讨论区提出来是先由管理者ping对应插件开发者,再由开发者解决问题)。

**Comment by ruibaby** [comment]
感谢你的建议，之前有考虑在 https://github.com/f2ccloud/ 组织下开启 Discussion 功能，专门用于接收这样的反馈，但考虑到 GitHub 访问问题和账号问题，可能后续会考虑在 lxware.cn 下单独开一个 Issue Tracker 的功能，或者使用三方的平台。

**Comment by QingFeng-awa** [comment]
4个月了,有后续吗

> 我个人觉得能跑起来Halo就代表有一定技术力了,因此对于GitHub账号和访问这种事情想必并不是什么很难的事情。\
> 如果凌霞内部还没考虑好/忘了/就没人提这件事不如就通过在f2c组织底下开讨论区作为过渡用。\
> 推荐一下[Apache Answer](https://answer.apache.org/zh-CN/),挺适合做Issue Tracker的,1Panel软件商店也有这个。

**Comment by ruibaby** [comment]
已经在 https://github.com/orgs/lxware-dev/discussions 开通 Discussions，后续会陆续在产品中引导到这里讨论。

感谢建议和关注。
